### DISCUSSION TITLE
Do we have any tool that integrates with Brakeman to manage the remediation program effectively?

### DISCUSSION THREAD

**Discussion by prakashsivakumars** [presidentbeef/brakeman#1690] [Q&A]
Hi, I have started using brakeman for my companies ruby on rails web application. I'm trying to see if there are any tools that integrates with brakeman and do the remediation and triaging effectively. Any help in this regard would be highly appreciated.

**Comment by presidentbeef** [comment]
Hi @prakashsivakumars -

There are some options out there.

Take a look at https://github.com/presidentbeef/brakeman/wiki/Tools-Using-Brakeman and https://github.com/presidentbeef/brakeman/wiki/Tools-Using-Brakeman

https://defectdojo.github.io/django-DefectDojo/ is another.

I'm sure there are other commercial options, too.
